app/controllers/redirection_controller.rb

```diff
@@ -1,28 +1,53 @@
 class RedirectionController < ApplicationController
   def announcements
-    parameters = { keywords: params['keywords'],
-                  level_one_taxon: params['taxons'].try(:first),
-                  level_two_taxon: params['subtaxons'].try(:first),
-                  people: params['people'],
-                  organisations: params['departments'],
-                  world_locations: params['world_locations'],
-                  public_timestamp: { from: params['from_date'], to: params['to_date'] }.compact.presence }
+    parameters = convert_common_parameters.merge(
+      level_two_taxon: params['subtaxons'].try(:first),
+      people: params['people'],
+      world_locations: params['world_locations']
+    )
     respond_to do |format|
       format.html { redirect_to(finder_path('search/news-and-communications', params: parameters)) }
       format.atom { redirect_to(finder_path('search/news-and-communications', params: parameters, format: :atom)) }
     end
   end
 
   def publications
-    parameters = { keywords: params['keywords'],
-                  level_one_taxon: params['taxons'].try(:first),
-                  level_two_taxon: params['subtaxons'].try(:first),
-                  organisations: params['departments'],
-                  world_locations: params['world_locations'],
-                  public_timestamp: { from: params['from_date'], to: params['to_date'] }.compact.presence }
+    parameters = convert_common_parameters.merge(
+      level_two_taxon: params['subtaxons'].try(:first),
+      world_locations: params['world_locations']
+    )
     respond_to do |format|
       format.html { redirect_to(finder_path('search/all', params: parameters)) }
       format.atom { redirect_to(finder_path('search/all', params: parameters, format: :atom)) }
     end
   end
+
+  def published_statistics
+    parameters = convert_common_parameters
+    respond_to do |format|
+      format.html { redirect_to(finder_path('search/statistics', params: parameters)) }
+      format.atom { redirect_to(finder_path('search/statistics', params: parameters, format: :atom)) }
```

Comment:
You can just use `convert_common_parameters` here, then you don't need to create a new variable 'parameters'.